server/container_attach.go

```diff
@@ -24,7 +26,7 @@ func (s *Server) Attach(ctx context.Context, req *pb.AttachRequest) (resp *pb.At
 func (s StreamService) Attach(containerID string, inputStream io.Reader, outputStream, errorStream io.WriteCloser, tty bool, resize <-chan remotecommand.TerminalSize) error {
 	c, err := s.runtimeServer.GetContainerFromShortID(containerID)
 	if err != nil {
-		return fmt.Errorf("could not find container %q: %v", containerID, err)
+		return status.Errorf(codes.NotFound, err.Error())
```

Comment:
The current message may still be appropriate, since we would want end users to know that over something lower level

Reply:
Yeah, cleaning up the messages :+1: